src/containers/Dashboard/ContentContainer.js

```diff
@@ -14,7 +14,7 @@ class ContentContainer extends Component {
           <div className="prac ml-5">
             <span>Practice</span>
           </div>
-          <div className="ml-5">
+          <div className="ml-5">.
```

Comment:
@Humnaa this full stop at end seems to be a typo.

Reply:
Yes , it was a typo and I have removed it